proto/asgt/dataservice/v1alpha1/scanned_invoice_data_service.proto

```diff
@@ -43,6 +43,7 @@ message ScannedInvoiceEntryInput {
 message CreateScannedInvoiceDatasetRequest {
   string dataset_name = 1;
   repeated string tags = 2;
+  repeated string targets = 5;
```

Comment:
Whats the reason you are adding it here? - To have strings in the same place? :)

Reply:
Just keeping the order that we have elsewhere in the code (consumer, name, type, tags, targets, ...)